app/dashboard/models.py

```diff
@@ -987,6 +990,17 @@ def additional_funding_summary_sentence(self):
 
         return sentence
 
+    @property
+    def reserved_for_user_handle(self):
+        if self.bounty_reserved_for_user:
+            return self.bounty_reserved_for_user.handle
+        return None
+
+    @reserved_for_user_handle.setter
+    def reserved_for_user_handle(self, handle):
+        profile = Profile.objects.filter(handle__iexact=handle).first()
```

Comment:
If no profile matches the provided handle, this will throw an unhandled exception.

Reply:
I wrapped it. Although ` Profile.objects.filter(handle__iexact='xyz').first()` returns None if nothing is found. 👍